src/run.ts

```diff
@@ -297,10 +290,15 @@ export async function applyRenderer(
 
         let data: string | Buffer
 
-        if (format === 'svg' || format === 'json') {
```

Comment:
@julien-deramond

Looks like you added this in https://github.com/antfu-collective/sponsorkit/pull/99. Would you mind double checking this is fine?

(We gotta start adding tests to this project)

Reply:
Seems like @julien-deramond has been busy lately, maybe we can merge this PR first?